### PR TITLE
Making event-bus examples CE spec 3.0 compliant

### PR DESCRIPTION
**Pull request by sayanh**
**Description**

Changes proposed in this pull request:

- Making event-bus examples  CE spec 3.0 compliant

**Related issue(s)**

See https://github.com/kyma-project/kyma/issues/4388

### CODE REVIEW

File: event-subscription/lambda/README.md [hunk 1]
```diff
@@ -94,6 +94,18 @@ kubeless get-server-config
         ```bash
         kubectl logs -f $(kubectl get po -n $KYMA_EXAMPLE_NS -l function=hello-with-data --no-headers | grep -i running | awk '{print $1}') -c hello-with-data -n $KYMA_EXAMPLE_NS
         ```
+6. To access attributes of events which are compliant to CloudEvents [specification](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md), you can access event object as follows:
```
Comment (bszwarc) [inline event-subscription/lambda/README.md]:
```suggestion
6. To access Event attributes compliant with the CloudEvents [specification](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md) use the following paths in the request:
```
Question: can we write "paths" here? How are the attributes accessed - is it a specific request which uses the path?

File: event-subscription/service/README.md [hunk 2]
```diff
@@ -81,6 +81,19 @@ Access the Event publishing API from the cluster through the `8080` port on the
     ```
     > **NOTE:** To send multiple Events, change the `orderCode` to have a unique value for each POST request.
 
+10. To access attributes of events which are compliant to CloudEvents [specification](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md), you can access event object as follows:
```
Comment (bszwarc) [inline event-subscription/service/README.md]:
```suggestion
10. To access Event attributes compliant with the CloudEvents [specification](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md) use the following paths in the request:
```

File: event-subscription/lambda/README.md (hunk 1 above)
Comment (sayanh) [reply event-subscription/lambda/README.md]: Yes we can and with more context.
`use the following paths for the event object in the request:`
Will it make sense?

**Comment by sayanh** [comment]
I need to close this as I created a branch in this repo by mistake. The PR which has this change is https://github.com/kyma-project/examples/pull/130/files

**Pull request closed without merging**